includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -216,7 +216,7 @@ public function column_customer( $item ) {
 	 */
 	public function column_cb( $item ) {
 		$customer = edd_get_customer_by( 'id', $item['customer_id'] );
-		$name     = isset( $customer->name ) ? $customer->name : $customer->email;
+		$name     = isset( $customer ) ? $customer->name : $item['id'];
```

Comment:
I think you want to leave the `isset` as it was (checking for the name value) and then as I read the issue, I think Ashley was suggesting that the item ID fallback be prefaced with "address ID {ID number here}"? So then the screen reader text will either read "Select John Doe" (which I'm thinking now isn't as clear as it could be for an address) or "Select address ID ##". Maybe the label needs to be rethought as well.

Reply:
More like this?